index.html

```diff
@@ -0,0 +1,180 @@
+            </div>
+            <div class="thumbnail">
+                <img src="https://img.youtube.com/vi/zUwB_imVjmg/maxresdefault.jpg" alt=""/>
+            </div>
+            <div class="details">
+                <div class="Author">
+                    <img src="https://yt3.ggpht.com/bpzY-S4DYlbTeOpY5hIA7qz_hcbMkgvLAugtwKBGTTImNnWAGudX0y53bo_fJZ0auypxrWkUiw=s88-c-k-c0x00ffffff-no-rj" alt="">
+                </div>
+                <div class="title"></div>
+                <h3>Introverts & content craetion | Seta Gawasker</h3>
+                <a href="">
+                    AbuTalha Lala
+                </a>
+                    <span> 2M Views . 3 Months Ago</span>
+            </div>
+            <div class="thumbnail">
+                <img src="https://img.youtube.com/vi/zUwB_imVjmg/maxresdefault.jpg" alt=""/>
+            </div>
+            <div class="details">
+                <div class="Author">
+                    <img src="https://yt3.ggpht.com/bpzY-S4DYlbTeOpY5hIA7qz_hcbMkgvLAugtwKBGTTImNnWAGudX0y53bo_fJZ0auypxrWkUiw=s88-c-k-c0x00ffffff-no-rj" alt="">
+                </div>
+                <div class="title"></div>
+                <h3>Introverts & content craetion | Seta Gawasker</h3>
+                <a href="">
+                    AbuTalha Lala
+                </a>
+                    <span> 2M Views . 3 Months Ago</span>
+            </div>
+            <div class="thumbnail">
+                <img src="https://img.youtube.com/vi/zUwB_imVjmg/maxresdefault.jpg" alt=""/>
+            </div>
+            <div class="details">
+                <div class="Author">
+                    <img src="https://yt3.ggpht.com/bpzY-S4DYlbTeOpY5hIA7qz_hcbMkgvLAugtwKBGTTImNnWAGudX0y53bo_fJZ0auypxrWkUiw=s88-c-k-c0x00ffffff-no-rj" alt="">
+                </div>
+                <div class="title"></div>
+                <h3>Introverts & content craetion | Seta Gawasker</h3>
+                <a href="">
+                    AbuTalha Lala
+                </a>
+                    <span> 2M Views . 3 Months Ago</span>
+            </div>
+        </div>
+
+        <div class="videos">
+            <div class="thumbnail">
+                <img src="https://img.youtube.com/vi/zUwB_imVjmg/maxresdefault.jpg" alt=""/>
+            </div>
+            <div class="details">
+                <div class="Author">
+                    <img src="https://yt3.ggpht.com/bpzY-S4DYlbTeOpY5hIA7qz_hcbMkgvLAugtwKBGTTImNnWAGudX0y53bo_fJZ0auypxrWkUiw=s88-c-k-c0x00ffffff-no-rj" alt="">
+                </div>
+                <div class="title"></div>
+                <h3>Introverts & content craetion | Seta Gawasker</h3>
+                <a href="">
+                    AbuTalha Lala
+                </a>
+                    <span> 2M Views . 3 Months Ago</span>
+            </div>
+            <div class="thumbnail">
+                <img src="https://img.youtube.com/vi/zUwB_imVjmg/maxresdefault.jpg" alt=""/>
+            </div>
```

Comment:
**optional**   it is suggested that you must take care of indentation and white spaces so that you code looks neat and clean and easily readable for humans.